lib/matplotlib/tests/README.rst

```diff
@@ -4,5 +4,5 @@ About Matplotlib Testing Infrastructure
 Information on the testing infrastructure is provided in
 the Testing section of the Matplotlib Developers’ Guide:
 
-* https://matplotlib.org/devel/testing.html
-* <mpl_source_dir>/doc/devel/coding_guide.rst (equivalent, but in reST format)
+- `Website <https://matplotlib.org/devdocs/devel/testing.html>`_
```

Comment:
I’m not convinced that ReST formatting is an advantage here. I assume that this file will be mostly opened as a source file in an editor, in which case plain URLs are easier to read than ReST formatted URLs.